tests/atomic_swap/test_get_info.py

```diff
@@ -0,0 +1,205 @@
+"""
+Provide tests for command line interface's atomic swap commands.
+"""
+import json
+import re
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    ADDRESS_REGEXP,
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_27_IN_TESTNET_ADDRESS,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+    SWAP_IDENTIFIER_REGEXP,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_swap_info():
+    """
+    Case: get information about atomic swap by its identifier.
+    Expect: information about swap is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'atomic-swap',
+        'get-info',
+        '--id',
+        '033402fe1346742486b15a3a9966eb5249271025fc7fb0b37ed3fdb4bcce6808',
+        '--node-url',
+        NODE_27_IN_TESTNET_ADDRESS,
+    ])
+
+    swap_info = json.loads(result.output).get('result').get('information')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert re.match(pattern=ADDRESS_REGEXP, string=swap_info.get('sender_address')) is not None
```

Comment:
Why you check just pattern? Check, please equality swap_id or if swap_id in data. Add/Replace more one checking.

Reply:
@Alladin9393, done.